src/modules/dummyqml/DummyQmlViewStep.h

```diff
@@ -19,20 +20,27 @@
 #ifndef DUMMYQMLVIEWSTEP_H
 #define DUMMYQMLVIEWSTEP_H
 
+#include <PluginDllMacro.h>
+#include "locale/TranslatableConfiguration.h"
```

Comment:
Yes, sure. It's shorter than declaring the class (which is all you'd need for that one pointer member).